Article Review

Single-Cell Genomics: A Stepping Stone for Future Immunology Discoveries

**Application of single-cell RNA sequencing in immunology**

Single-cell genomics can be used to characterize the functional diversity of immune cell types and their highly flexible identities. The use of single-cell RNA sequencing in immunology research can help overcome current technological limitations, one way being the elimination of the need for predefined markers in cell type breakdown of complex tissues. Advances in single-cell genomics that further enable its use in immunology research include:

- Improved reproducibility, throughput, and sensitivity via more efficient cell isolation methods, automation, and microfluidics techniques
- Noise reduction via implementation of unique molecular identifiers
- New algorithms for clustering, visualization, and modeling that are specialized for handling single-cell RNA sequencing data

These advances allow accurate and reproducible isolation, measurement, and modeling of gene expression profiles from hundreds of thousands of cells.

Single-cell genomics studies on lymphoid organs have unveiled the functional heterogeneity of immune cell types that are typically considered homogeneous. It has been reported that classically defined immune entities can consist of different cell populations with overlapping markers. For example, single-cell profiling of hematopoietic progenitor populations revealed a large degree of functional heterogeneity and commitment to various lineages. Another example is the discovery of disease-associated macroglia (DAM), which was made possible by single-cell analysis. DAMs are molecularly distinct microglia cells that interact with and phagocytize plaques in Alzheimer's disease.

**Transcriptional sorting: from unbiased sampling to relevant cell types**

A new methodology must be devised to tackle the complexity of the immune system with the use of single-cell genomics. Optimization of tissue homogenization and/or fixation is necessary to preserve _in situ_ cell states and original cell composition. In many cases, current tissue homogenization protocols are optimized for bulk RNA sequencing or protein labeling, thus further calibration may be needed to maximize the extraction of different cell types while retaining the original gene expression levels.

There are two main approaches for measuring single-cell transcriptomes: full transcriptome coverage and molecular counting methods. Full-coverage approaches are optimal for integrating mutations and/or inferring splicing events in single cells, while counting methods are based on sequencing of the 5' or 3' end of the molecule, and are suited for cost-effective single-cell profiling.

**Challenges and future developments**

One of the current challenges in the use of single-cell genomics in immunology is coping with large datasets of unbiased sample cells that encompass a large variety of cell types and cell states. Such analysis requires successfully assigning each cell type into accurate classifications, including subtle distinctions within cell types (for example, various T helper subsets or different dendritic cell types), while allowing the characterization of shared modular transcriptional programs (such as proliferation, inflammation, MHC-II pathway, exhaustion, or anti-viral response). Recent analytics have begun to tackle this challenge, and further advances will enable a detailed immune "table of elements", including all immune types and states that can be used as a reference.

Another challenge is the lack of vital contextual information regarding space, clonality, time, metabolic processes, and epigenetic state in single-cell RNA sequencing maps following tissue homogenization. Future developments aim to simultaneously record and integrate complementary types of cellular and molecular information from the same cell. One example is the addition of sophisticated barcoded transcribed RNA molecules to record single-cell genetic perturbation.

**Translational immunology**

The full potential of single-cell genomics to revolutionize immunology has yet to be realized. Lowering costs and constant improvements in single-cell technologies, combined with standardization and accessibility efforts, will transform single-cell genomics into a common instrument for immunology research widely used in academia, the biotech industry, and the clinic. New computational tools are also being developed for the standardization, accumulation, storage, and accessibility of single-cell data, as well as comparison of current and future datasets, all while controlling for immense immunological variation between individuals. It is possible that in the future, numerous patients of various immune-related pathologies will be profiled with single-cell sequencing, allowing for better patient classification, identification of novel biomarkers, prediction of drug responses, and identification of novel cell targets and pathways. This will rise to new immunotherapies that are more efficient, less harmful, and more precise.

**References**

Giladi, A., & Amit, I. (2018). Single-Cell Genomics: A Stepping Stone for Future Immunology Discoveries. _Cell_, 172(1), 14-21. <https://doi.org/10.1016/j.cell.2017.11.011>
